### PR TITLE
Add desklet visibility toggling to the Show Desktop applet context menu, update with class syntax

### PR DESCRIPTION
**Pull request by jaszhix**

### CODE REVIEW

File: files/usr/share/cinnamon/applets/show-desktop@cinnamon.org/applet.js [hunk 1]
```diff
@@ -33,13 +31,21 @@ MyApplet.prototype = {
 
         this.set_applet_icon_name("user-desktop");
         this.set_applet_tooltip(_("Show desktop"));
-    },
 
-    on_applet_removed_from_panel: function() {
+        this.showDeskletsOption = new PopupMenu.PopupIconMenuItem(
+            _('Show Desklets'),
```
Comment (NikoKrause) [inline files/usr/share/cinnamon/applets/show-desktop@cinnamon.org/applet.js]: _("Show Desklets")

File: files/usr/share/cinnamon/applets/show-desktop@cinnamon.org/applet.js [hunk 2]
```diff
@@ -108,9 +114,18 @@ MyApplet.prototype = {
         }
         this._did_peek = false;
     }
+
+    toggleShowDesklets() {
+        if (Main.deskletContainer.isModal) {
+            Main.deskletContainer.lower();
+            this.showDeskletsOption.label.set_text(_('Show Desklets'));
```
Comment (NikoKrause) [inline files/usr/share/cinnamon/applets/show-desktop@cinnamon.org/applet.js]: _("Show Desklets")

File: files/usr/share/cinnamon/applets/show-desktop@cinnamon.org/applet.js [hunk 3]
```diff
@@ -108,9 +114,18 @@ MyApplet.prototype = {
...
+        } else {
+            Main.deskletContainer.raise();
+            this.showDeskletsOption.label.set_text(_('Hide Desklets'));
```
Comment (NikoKrause) [inline files/usr/share/cinnamon/applets/show-desktop@cinnamon.org/applet.js]: _("Hide Desklets")

**Comment by NikoKrause** [comment]
Not sure if it's still the case, but in the past the makepot script didn't recognized single quotes. So all the default applets use double quotes for translatable strings.

**Comment by jaszhix** [comment]
Ah, I think it should work because we passed JavaScript as the language to [xgettext](https://github.com/linuxmint/Cinnamon/blob/5a9de050d453140340fd0424811a751f367015ef/makepot#L8).

**Pull request merged**